resources/views/community/components/user/profile/player-stats.blade.php

```diff
@@ -32,71 +24,70 @@ class="relative w-full px-2 pt-2 bg-embed rounded mb-6 pb-4 transition-all"
-            <x-user.profile.mode-player-stat-cells
-                :averageFinishedGames="$averageFinishedGames"
-                :hardcoreRankMeta="$hardcoreRankMeta"
-                :mode="$primaryMode"    
-                :retroRatio="$retroRatio"
-                :softcoreRankMeta="$softcoreRankMeta"
-                :totalHardcoreAchievements="$totalHardcoreAchievements"
-                :totalSoftcoreAchievements="$totalSoftcoreAchievements"
-                :userMassData="$userMassData"
+    @if ($hasMixedProgress)
+        @if ($primaryMode === 'hardcore')
+            <x-user.profile.arranged-stat-items
+                :stats="[
+                    $playerStats['hardcorePointsStat'],                 $playerStats['softcorePointsStat'],
+                    $playerStats['hardcoreSiteRankStat'],               $playerStats['softcoreSiteRankStat'],
+                    $playerStats['hardcoreAchievementsUnlockedStat'],   $playerStats['softcoreAchievementsUnlockedStat'],
+                    $playerStats['retroRatioStat'],                     $playerStats['startedGamesBeatenPercentageStat'],
+                ]"
             />
-        </div>
-
-        <div class="flex flex-col gap-y-1">
-            <x-user.profile.mode-player-stat-cells
-                :averageFinishedGames="$averageFinishedGames"
-                :hardcoreRankMeta="$hardcoreRankMeta"
-                :mode="$secondaryMode"    
-                :retroRatio="$retroRatio"
-                :softcoreRankMeta="$softcoreRankMeta"
-                :totalHardcoreAchievements="$totalHardcoreAchievements"
-                :totalSoftcoreAchievements="$totalSoftcoreAchievements"
-                :userMassData="$userMassData"
+        @elseif ($primaryMode === 'softcore')
+            <x-user.profile.arranged-stat-items
+                :stats="[
+                    $playerStats['softcorePointsStat'],                 $playerStats['hardcorePointsStat'],
+                    $playerStats['softcoreSiteRankStat'],               $playerStats['hardcoreSiteRankStat'],
+                    $playerStats['softcoreAchievementsUnlockedStat'],   $playerStats['hardcoreAchievementsUnlockedStat'],
+                    $playerStats['startedGamesBeatenPercentageStat'],   $playerStats['retroRatioStat'],
+                ]"
             />
-        </div>
-    </div>
+        @endif
+    @elseif ($primaryMode === 'hardcore')
+        <x-user.profile.arranged-stat-items
+            :stats="[
+                $playerStats['hardcoreAchievementsUnlockedStat'],   $playerStats['retroRatioStat'],
+                $playerStats['totalGamesBeatenStat'],               $playerStats['startedGamesBeatenPercentageStat'],   
+            ]"
+        />
+    @elseif ($primaryMode === 'softcore')
+        <x-user.profile.arranged-stat-items
+            :stats="[
+                $playerStats['softcoreAchievementsUnlockedStat'],   $playerStats['startedGamesBeatenPercentageStat'], 
```

Comment:
I assume it's intentional to keep things in matching rows, but Total Games Beaten doesn't show for pure-softcore players.

Reply:
Right now it's coming purely from `PlayerStat`, which drives the beaten game leaderboard and only takes hardcore numbers into account.

```php
// Total games beaten
$totalGamesBeaten = (int) PlayerStat::where('user_id', $user->ID) // keep in mind, this is hardcore only.
    ->where('system_id', null)
    ->sum('value');
$totalGamesBeatenStat = [
    'label' => 'Total games beaten',
    'value' => localized_number($totalGamesBeaten),
    'isMuted' => !$totalGamesBeaten,
    // TODO: href to 'ranking.beaten-games' for a specific user.
];
```